plugins/dicom_viewer/plugin.json

```diff
@@ -7,7 +7,7 @@
         "dependencies": {
             "daikon": "^1.2.15",
             "shader-loader": "1.3.0",
-            "vtk.js": "2.24.1"
+            "vtk.js": "5.10.3"
```

Comment:
Did you verify that the viewer is still working properly under this version?

Reply:
Yes, tested it locally.

Reply:
It worked for me as well.